src/components/MySquads.js

```diff
@@ -102,6 +102,7 @@ const MySquads = ({ userID }) => {
       </Modal>
       <FlatList
         data={userSquads}
+        contentContainerStyle={{ paddingBottom: 200 }}
```

Comment:
This is all we needed to fix that scrolling issue I mentioned.

Reply:
🥇 Nice!